src/api/pastel-rpc/transactions.ts

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Please do not disable eslint

Reply:
@br4in3x So what happens if I need to use `any`?

Reply:
I have removed `any`